kitchen/config.yml

```diff
@@ -0,0 +1,12 @@
+---
+driver_plugin: docker
+driver_config:
+  provision_command:
+    - curl -L https://www.opscode.com/chef/install.sh | bash -s -- -v 12.0.3
```

Comment:
This installs an outdated chef version. And it pins the version. I'd prefer to just install the latest as default.